libs/hmpb/layout/src/layout.ts

```diff
@@ -360,6 +361,32 @@ export function TimingMarkGrid({ m }: { m: Measurements }): AnyElement {
   };
 }
 
+// Special case hack for NH school election ballots (which need to be visually
+// differentiated from town election ballots but also need to be scanned on the
+// same scanner, and therefore must use the same election definition). if a
+// ballot style only has a single district that appears to correspond to a
+// school district, then change the election title. In the future, we might
+// support customization like this by allowing custom content for different
+// precinct splits or by using different ballot templates.
+function modifyElectionTitleForNhSchoolElections(
```

Comment:
This name sounds like it's going to mutate its arguments, but it actually just extracts the election title with a special case for NH school elections. Maybe it should be called something like `getElectionTitleOrSchoolElectionTitle`?

Reply:
Updated: 1cc25ab16